tide/core/data/FFMPEGMovie.cpp

```diff
@@ -177,59 +166,52 @@ double FFMPEGMovie::getFrameDuration() const
     return _videoStream->getFrameDuration();
 }
 
-TextureFormat FFMPEGMovie::getFormat() const
-{
-    return _format;
-}
-
-void FFMPEGMovie::setFormat(const TextureFormat format)
-{
-    _format = format;
-}
-
 PicturePtr FFMPEGMovie::getFrame(double posInSeconds)
 {
     posInSeconds = std::max(0.0, std::min(posInSeconds, getDuration()));
+    const auto frameDuration = _videoStream->getFrameDuration();
+    const auto target = std::max(0.0, posInSeconds - frameDuration);
+    const auto frameIndex = _videoStream->getFrameIndex(target);
+
+    _frameIndex = frameIndex;
+    _streamPosition = posInSeconds;
```

Comment:
not needed

Reply:
It is actually used by FFMPEGMovie::getPosition()